proxy/src/telemetry/metrics/latency.rs

```diff
@@ -105,22 +107,22 @@ impl Histogram {
         // 18,446,744,073,709,551,615 is the maximum 64-bit unsigned integer.
         //      1.8446744073709551615e19 / 5.184e16 = 355 (about 1 year)
         //
-        // So at 1 million RPS with 1-minute latencies, the sum will overflow
+        // So at 1 million RPS with 1-minute latencies, the sum will wrap
         // in about a year. We don't really expect a conduit-proxy process to
         // run that long (or see this kind of load), but we can revisit this
         // if supporting extremely long-running deployments becomes a priority.
         //
         // (N.B. that storing latencies in whole milliseconds rather than tenths
         // of milliseconds would change the time to overflow to almost 10
         // years.)
-        self.sum += measurement.0 as u64;
+        self.sum += Wrapping(measurement.0 as u64);
```

Comment:
is there any reason this can't just be `sum.wrapping_add`? what does the newtype buy us?

Reply:
also, i don't think the long explanatory comment is really necessary anymore.  we should just document (at a higher level) that counters wrap.

Reply:
@olix0r, personally, I prefer the  `Wrapping` type because it makes it explicit that "operations on this value *always* use wrapping arithmetic" (i.e. something I expect to never overflow can't overflow because I wasn't thinking and used regular arithmetic by accident somewhere).

Reply:
(will remove the big comment though)